### PR TITLE
kong: bump Kong to 1.1.0rc2

### PR DESCRIPTION
**Pull request by hutchic**

### CODE REVIEW

**Comment by yosifkit** [comment]
<details>
<summary>Diff:</summary>

```diff
diff --git a/_bashbrew-arches b/_bashbrew-arches
index 68b3080..0a3dd77 100644
--- a/_bashbrew-arches
+++ b/_bashbrew-arches
@@ -11,4 +11,6 @@ kong:0.15-centos @ amd64
 kong:1.0-centos @ amd64
 kong:1.1rc1 @ amd64
 kong:1.1rc1-centos @ amd64
+kong:1.1rc2 @ amd64
+kong:1.1rc2-centos @ amd64
 kong:latest @ amd64
diff --git a/_bashbrew-list b/_bashbrew-list
index 7b0da81..6d08a4c 100644
--- a/_bashbrew-list
+++ b/_bashbrew-list
@@ -34,7 +34,13 @@ kong:1.0.3-centos
 kong:1.1rc1
 kong:1.1rc1-alpine
 kong:1.1rc1-centos
+kong:1.1rc2
+kong:1.1rc2-alpine
+kong:1.1rc2-centos
 kong:1.1.0rc1
 kong:1.1.0rc1-alpine
 kong:1.1.0rc1-centos
+kong:1.1.0rc2
+kong:1.1.0rc2-alpine
+kong:1.1.0rc2-centos
 kong:latest
diff --git a/kong_1.0-centos/Dockerfile b/kong_1.1rc2-centos/Dockerfile
similarity index 97%
copy from kong_1.0-centos/Dockerfile
copy to kong_1.1rc2-centos/Dockerfile
index c792f71..2eca4c2 100644
--- a/kong_1.0-centos/Dockerfile
+++ b/kong_1.1rc2-centos/Dockerfile
@@ -1,7 +1,7 @@
 FROM centos:7
 LABEL maintainer="Kong Core Team <team-core@konghq.com>"
 
-ENV KONG_VERSION 1.0.3
+ENV KONG_VERSION 1.1.0rc2
 
 ARG SU_EXEC_VERSION=0.2
 ARG SU_EXEC_URL="https://github.com/ncopa/su-exec/archive/v${SU_EXEC_VERSION}.tar.gz"
diff --git a/kong_1.1rc1-centos/docker-entrypoint.sh b/kong_1.1rc2-centos/docker-entrypoint.sh
similarity index 100%
copy from kong_1.1rc1-centos/docker-entrypoint.sh
copy to kong_1.1rc2-centos/docker-entrypoint.sh
diff --git a/kong_0.15/Dockerfile b/kong_1.1rc2/Dockerfile
similarity index 88%
copy from kong_0.15/Dockerfile
copy to kong_1.1rc2/Dockerfile
index 3a9e5e6..6d09de0 100644
--- a/kong_0.15/Dockerfile
+++ b/kong_1.1rc2/Dockerfile
@@ -1,8 +1,8 @@
 FROM alpine:3.6
 LABEL maintainer="Kong Core Team <team-core@konghq.com>"
 
-ENV KONG_VERSION 0.15.0
-ENV KONG_SHA256 f2adc4e40a4a33c657955d6d9ec034d80827f915bb9fe8c2ac2a36aed7edf13b
+ENV KONG_VERSION 1.1.0rc2
+ENV KONG_SHA256 a4e2c7aead3ddb4e1443fd5f7d42e22caa251a8c46937b01eaae50fd8655d528
 
 RUN adduser -Su 1337 kong \
 	&& mkdir -p "/usr/local/kong" \
diff --git a/kong_1.1rc1-centos/docker-entrypoint.sh b/kong_1.1rc2/docker-entrypoint.sh
similarity index 100%
copy from kong_1.1rc1-centos/docker-entrypoint.sh
copy to kong_1.1rc2/docker-entrypoint.sh
```

</details>

File: library/kong [hunk 1]
```diff
@@ -2,6 +2,17 @@ Maintainers: Kong Docker Maintainers <support@konghq.com> (@thekonginc)
...
+GitFetch: refs/tags/1.1.0rc2
+Directory: alpine
+
+Tags: 1.1.0rc2-centos, 1.1rc2-centos
+GitCommit: 764155275f26b1447ce34f9255d2564d4fdd08a1
+GitFetch: refs/tags/1.1.0rc2
+Constraints: !aufs
+Directory: centos
+
 Tags: 1.1.0rc1-alpine, 1.1rc1-alpine, 1.1.0rc1, 1.1rc1
```
Comment (yosifkit) [inline library/kong]: Did you want to drop the `rc1` versions now that you are adding `rc2`?

Comment (hutchic) [reply library/kong]: At this time we'll keep it around 👍

**Comment by tianon** [comment]
LGTM (please do plan to remove these once the RCs are no longer considered "supported"; removing them here will remove them from the "Supported" section on the Hub readme and will prevent us from spending cycles rebuilding them on the official build server, but the tag will still be available to users who want it; see https://github.com/docker-library/official-images#library-definition-files for more detail on this.)

Build test of #5586; 6ed9adfc; `amd64` (`kong`):

```console

$ bashbrew build kong:1.1.0rc2-alpine
Using bashbrew/cache:5712e6864dbb6b22bbf9005c0bd481cf1cbd07e8936d8ef0f4739ad61b5425e8 (kong:1.1.0rc2-alpine)
Tagging kong:1.1.0rc2-alpine
Tagging kong:1.1rc2-alpine
Tagging kong:1.1.0rc2
Tagging kong:1.1rc2

$ test/run.sh kong:1.1.0rc2-alpine
testing kong:1.1.0rc2-alpine
	'utc' [1/4]...passed
	'cve-2014--shellshock' [2/4]...passed
	'no-hard-coded-passwords' [3/4]...passed
	'override-cmd' [4/4]...passed


$ bashbrew build kong:1.1.0rc2-centos
Using bashbrew/cache:ec9b695a3174c4f3b7b52d35d7d4c267e59d4e119ba6a6cacc3ee8f98385afa7 (kong:1.1.0rc2-centos)
Tagging kong:1.1.0rc2-centos
Tagging kong:1.1rc2-centos

$ test/run.sh kong:1.1.0rc2-centos
testing kong:1.1.0rc2-centos
	'utc' [1/4]...passed
	'cve-2014--shellshock' [2/4]...passed
	'no-hard-coded-passwords' [3/4]...warning: garbage password detected for 'root': 'locked'
passed
	'override-cmd' [4/4]...passed


$ bashbrew build kong:1.1.0rc1-alpine
Using bashbrew/cache:677b1fac10f1f87f7623b02e380b7eedcac7329b19f3eae702574c1b7fe6b251 (kong:1.1.0rc1-alpine)
Tagging kong:1.1.0rc1-alpine
Tagging kong:1.1rc1-alpine
Tagging kong:1.1.0rc1
Tagging kong:1.1rc1

$ test/run.sh kong:1.1.0rc1-alpine
testing kong:1.1.0rc1-alpine
	'utc' [1/4]...passed
	'cve-2014--shellshock' [2/4]...passed
	'no-hard-coded-passwords' [3/4]...passed
	'override-cmd' [4/4]...passed


$ bashbrew build kong:1.1.0rc1-centos
Using bashbrew/cache:2e87b6bde76a4b0b1beaca2e351aa5401ae83b8b896b06a452909da0b630d6f5 (kong:1.1.0rc1-centos)
Tagging kong:1.1.0rc1-centos
Tagging kong:1.1rc1-centos

$ test/run.sh kong:1.1.0rc1-centos
testing kong:1.1.0rc1-centos
	'utc' [1/4]...passed
	'cve-2014--shellshock' [2/4]...passed
	'no-hard-coded-passwords' [3/4]...warning: garbage password detected for 'root': 'locked'
passed
	'override-cmd' [4/4]...passed


$ bashbrew build kong:1.0.3-alpine
Using bashbrew/cache:792b95f26b5af9156c62e70ffcb6a47cd2861e0803fbf7f740c6545fd70c8276 (kong:1.0.3-alpine)
Tagging kong:1.0.3-alpine
Tagging kong:1.0.3
Tagging kong:1.0
Tagging kong:latest

$ test/run.sh kong:1.0.3-alpine
testing kong:1.0.3-alpine
	'utc' [1/4]...passed
	'cve-2014--shellshock' [2/4]...passed
	'no-hard-coded-passwords' [3/4]...passed
	'override-cmd' [4/4]...passed


$ bashbrew build kong:1.0.3-centos
Using bashbrew/cache:78db53824c405d76054a630ae2df8ae97e71447c9dbf1d75030d436c06dae6a0 (kong:1.0.3-centos)
Tagging kong:1.0.3-centos
Tagging kong:1.0-centos

$ test/run.sh kong:1.0.3-centos
testing kong:1.0.3-centos
	'utc' [1/4]...passed
	'cve-2014--shellshock' [2/4]...passed
	'no-hard-coded-passwords' [3/4]...warning: garbage password detected for 'root': 'locked'
passed
	'override-cmd' [4/4]...passed


$ bashbrew build kong:0.15.0-alpine
Using bashbrew/cache:1c629e7533b38aaf8c6f13e6abe8f8720282f911ca897954785b0f5a3bbc3427 (kong:0.15.0-alpine)
Tagging kong:0.15.0-alpine
Tagging kong:0.15-alpine
Tagging kong:0.15.0
Tagging kong:0.15

$ test/run.sh kong:0.15.0-alpine
testing kong:0.15.0-alpine
	'utc' [1/4]...passed
	'cve-2014--shellshock' [2/4]...passed
	'no-hard-coded-passwords' [3/4]...passed
	'override-cmd' [4/4]...passed


$ bashbrew build kong:0.15.0-centos
Using bashbrew/cache:683398cc7a2bb891fd8c43b1eedf70a84043b435006a818d4d0dc5157ece8770 (kong:0.15.0-centos)
Tagging kong:0.15.0-centos
Tagging kong:0.15-centos

$ test/run.sh kong:0.15.0-centos
testing kong:0.15.0-centos
	'utc' [1/4]...passed
	'cve-2014--shellshock' [2/4]...passed
	'no-hard-coded-passwords' [3/4]...warning: garbage password detected for 'root': 'locked'
passed
	'override-cmd' [4/4]...passed


$ bashbrew build kong:0.14.1-alpine
Using bashbrew/cache:fc7650ea767815ce9671200caa97c1f1d6a3b8345855d97292a8050af482db24 (kong:0.14.1-alpine)
Tagging kong:0.14.1-alpine
Tagging kong:0.14-alpine
Tagging kong:0.14.1
Tagging kong:0.14

$ test/run.sh kong:0.14.1-alpine
testing kong:0.14.1-alpine
	'utc' [1/4]...passed
	'cve-2014--shellshock' [2/4]...passed
	'no-hard-coded-passwords' [3/4]...passed
	'override-cmd' [4/4]...passed


$ bashbrew build kong:0.14.1-centos
Using bashbrew/cache:ca3757baa854ddfca8c023ec830bf27e67a503046807cd543d6b7e2a9d563eb5 (kong:0.14.1-centos)
Tagging kong:0.14.1-centos
Tagging kong:0.14-centos

$ test/run.sh kong:0.14.1-centos
testing kong:0.14.1-centos
	'utc' [1/4]...passed
	'cve-2014--shellshock' [2/4]...passed
	'no-hard-coded-passwords' [3/4]...warning: garbage password detected for 'root': 'locked'
passed
	'override-cmd' [4/4]...passed


$ bashbrew build kong:0.13.1-alpine
Using bashbrew/cache:063473761d20d041339600aa93fa7086bbfcff6bacf2363cb73f02eb8a768f00 (kong:0.13.1-alpine)
Tagging kong:0.13.1-alpine
Tagging kong:0.13-alpine
Tagging kong:0.13.1
Tagging kong:0.13

$ test/run.sh kong:0.13.1-alpine
testing kong:0.13.1-alpine
	'utc' [1/4]...passed
	'cve-2014--shellshock' [2/4]...passed
	'no-hard-coded-passwords' [3/4]...passed
	'override-cmd' [4/4]...passed


$ bashbrew build kong:0.13.1-centos
Using bashbrew/cache:ed9d3f557bbc05f4abca15d8a6d25c69bb36451bfaaa776aebbb99f62b73fadc (kong:0.13.1-centos)
Tagging kong:0.13.1-centos
Tagging kong:0.13-centos

$ test/run.sh kong:0.13.1-centos
testing kong:0.13.1-centos
	'utc' [1/4]...passed
	'cve-2014--shellshock' [2/4]...passed
	'no-hard-coded-passwords' [3/4]...warning: garbage password detected for 'root': 'locked'
passed
	'override-cmd' [4/4]...passed


$ bashbrew build kong:0.12-alpine
Using bashbrew/cache:e24dda1bb4b34129115bbf6d7299a4ce2c795a40ee19ffbec98c67d50c35a1fc (kong:0.12-alpine)
Tagging kong:0.12-alpine
Tagging kong:0.12.3-alpine

$ test/run.sh kong:0.12-alpine
testing kong:0.12-alpine
	'utc' [1/4]...passed
	'cve-2014--shellshock' [2/4]...passed
	'no-hard-coded-passwords' [3/4]...passed
	'override-cmd' [4/4]...passed


$ bashbrew build kong:0.12-centos
Using bashbrew/cache:a2e9e776b1455f5af46acde9ef0dd880ebf923c61d032d54faed7e5916b6f409 (kong:0.12-centos)
Tagging kong:0.12-centos
Tagging kong:0.12.3-centos
Tagging kong:0.12
Tagging kong:0.12.3

$ test/run.sh kong:0.12-centos
testing kong:0.12-centos
	'utc' [1/4]...passed
	'cve-2014--shellshock' [2/4]...passed
	'no-hard-coded-passwords' [3/4]...warning: garbage password detected for 'root': 'locked'
passed
	'override-cmd' [4/4]...passed


$ bashbrew build kong:0.11-alpine
Using bashbrew/cache:990dd69825449bba03d5e7bdc3a010fe0f129ca1546186a42811b7cbc0981f83 (kong:0.11-alpine)
Tagging kong:0.11-alpine
Tagging kong:0.11.2-alpine

$ test/run.sh kong:0.11-alpine
testing kong:0.11-alpine
	'utc' [1/4]...passed
	'cve-2014--shellshock' [2/4]...passed
	'no-hard-coded-passwords' [3/4]...passed
	'override-cmd' [4/4]...passed


$ bashbrew build kong:0.11
Using bashbrew/cache:af7169e220ae0fe381334dba566f67c5148ce1d13a721ffdc36c28f57652e4c6 (kong:0.11)
Tagging kong:0.11
Tagging kong:0.11.2

$ test/run.sh kong:0.11
testing kong:0.11
	'utc' [1/4]...passed
	'cve-2014--shellshock' [2/4]...passed
	'no-hard-coded-passwords' [3/4]...warning: garbage password detected for 'root': 'locked'
passed
	'override-cmd' [4/4]...passed

```

**Pull request merged**